_pages/openid-connect.md

```diff
@@ -137,6 +137,8 @@ https://idp.int.login.gov/openid_connect/authorize?
    - `address`
    - `email`
    - `phone`
+   - `profile:birthdate`
```

Comment:
This was me matching the openid connect `birthdate` attribute, so it made sense to me to ask for `birthdate` and get `birthdate` back?